tools/notification-configuration/github-codeowner-subscriber/GitHubNameResolver.cs

```diff
@@ -1,94 +0,0 @@
-﻿using Kusto.Data;
```

Comment:
This file being deleted seems a little like a bad merge with your other change, might want to make sure it is correct.

Reply:
The file not being deleted was an error with the other merge. This compiles just fine/isn't referenced in code.